src/libraries/Directory.Build.targets

```diff
@@ -64,6 +64,9 @@
     <BinPlaceRef Condition="'$(BinPlaceRef)' == '' and ('$(IsReferenceAssembly)' == 'true' or '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRef>
     <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' and ('$(IsRuntimeAssembly)' == 'true' or '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRuntime>
     <BinPlaceForTargetVertical Condition="'$(BinPlaceForTargetVertical)' == ''">true</BinPlaceForTargetVertical>
+
+    <!-- All IsNETCoreApp assemblies will get binplaced to the test sharedframework after the ILLink step. -->
+    <BinPlaceTestSharedFramework Condition="'$(IsNETCoreApp)' == 'true' and '$(BuildAllProjects)' == 'true'">false</BinPlaceTestSharedFramework>
```

Comment:
I think we agreed that we're going to continue binplacing the test shared framework from each individual assembly build, not the whole trimmed shared framework because that would conflict with inner developer experience.

Reply:
Yes - will remove this change and `CopyILTrimmedSharedFrameworkLibraries` above.